introduction.md

```diff
@@ -11,3 +11,7 @@ Ruby, JavaScript, HTML & CSS, iOS or Electives
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?